.github/workflows/release.yml

```diff
@@ -68,7 +68,9 @@ jobs:
         run: |
           echo SDKROOT=$(xcrun --sdk macosx --show-sdk-path) >> $GITHUB_ENV
 
-      - run: go build -v -o ${{ matrix.target.goos }}-${{ matrix.target.goarch }} .
+      - run: |
+          semver=$(git describe --tags | grep '^v' | head -1) 
```

Comment:
Since it's got the suffix at the end, it's not really a semver, so we should just call it `version` or something.

Reply:
I just realised there is a ${{ needs.release.outputs.version }} in that file. Forest for the trees I swear! We should just use that

Reply:
https://github.com/samuong/alpaca/pull/81/commits/2da2d0be24e4d06e5ce6ce08ef3d3101af844015

Reply:
If you want current local commit that is a new requirement. This satisfies the version number requirement for #11